manifests/params.pp

```diff
@@ -242,6 +242,10 @@
   $logs           = true
   $logs_listen_on = 'https'
 
+  # HTTPBoot settings - requires optional httpboot puppet module
+  $httpboot           = undef
+  $httpboot_listen_on = 'https'
```

Comment:
We need to listen by default on both, actually HTTP is more likely to be used because HTTPS requires users to enroll X509 certificate into firmware so I expects more users to simply use HTTP.

Reply:
Updated. Thanks!